tests/testthat/test-biology.R

```diff
@@ -23,7 +23,10 @@ test_that('total_M and EIR functions are consistent with equilibrium EIR', {
   })
 
   expect_equal(
-    actual_EIR,
+    actual_EIR * 0.766,
```

Comment:
It seems cleaner to remove omega from above. Pre-calculated values make tests quite brittle.

Reply:
Sure thing! Have done :)